raft-tech/TANF-app
# Feat/1965 approved user homepage

## Summary of Changes
_Provide a brief summary of changes_
Pull request closes #1965

<img width="1439" alt="Screen Shot 2023-03-03 at 12 46 12 PM" src="https://user-images.githubusercontent.com/7119690/222826425-6e43b98b-b0cf-41ba-8574-15811be60b26.png">

## How to Test
_List the steps to test the PR_
These steps are generic, please adjust as necessary.
```
cd tdrs-frontend && docker-compose -f docker-compose.yml -f docker-compose.local.yml up -d
cd tdrs-backend && docker-compose -f docker-compose.yml -f docker-compose.local.yml up -d 
```

1. Open http://localhost:3000/ and sign in.
2. Got to home page and test the resource card buttons.
3. Check styling is consistent with resource card on signing page.

## Deliverables
_More details on how deliverables herein are assessed included [here](../docs/How-We-Work/our-priorities-values-expectations.md#Deliverables)._

### [Deliverable 1: Accepted Features](../docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-1-Accepted-Features)
**Acceptance Criteria:** 
- [x] Update "Next Reporting Deadline" to "Data Reporting Deadlines"
- [x] Reporting deadline table presented, see reference table in supporting documentation
- [x] Resource cards are present on "Home"
- [x] User permissions are no longer visible

**Tasks:**
- [x] Hardcode the new deadline content, display the entire reporting deadline table
- [x] If possible, reference the resource cards content with an abstraction instead of duplication
- [x] Remove the permissions listing

### [Deliverable 2: Tested Code](../docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-2-Tested-Code)

+ Are all areas of code introduced in this PR meaningfully tested?
+ Are code coverage minimums met?

### [Deliverable 3: Properly Styled Code](../docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-3-Properly-Styled-Code)


### [Deliverable 4: Accessible](../docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-4-Accessibility)



### [Deliverable 5: Deployed](../docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-5-Deployed)


### [Deliverable 6: Documented](../docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-6-Code-documentation)


### [Deliverable 7: Secure](../docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-7-Secure)


### [Deliverable 8: User Research](../docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-8-User-Research)

Research product(s) clearly articulate(s):
